extensions/data-workspace/package.json

```diff
@@ -114,13 +118,22 @@
         },
         {
           "command": "projects.openExisting"
+        },
+        {
+          "command": "projects.manageProject",
+          "when": "false"
         }
       ],
       "view/item/context": [
         {
           "command": "projects.removeProject",
           "when": "view == dataworkspace.views.main && viewItem == databaseProject.itemType.project",
           "group": "9_dbProjectsLast@9"
+        },
+        {
+          "command": "projects.manageProject",
+          "when": "view == dataworkspace.views.main && viewItem == databaseProject.itemType.project",
+          "group": "1_dbProjectsFirst@0"
```

Comment:
can you change this (and the one above) to be projects, not dbProjects?

Reply:
I modified them and this what the right click options looked like:
![image](https://user-images.githubusercontent.com/57200045/107157777-dbb4a200-693a-11eb-9d1c-2379dbce8994.png)

Manage and remove project moved to their own group.
So, I updated the manage to be without db (and for it to be on top, provided 0 as the starting number for group), and also removed the viewItem to be only databaseProject, because other projects can have the dashboard as well.
And keeping dbProjects for Remove project, so that it stays in the same group.
![image](https://user-images.githubusercontent.com/57200045/107157887-6d241400-693b-11eb-8f58-cc5a54301628.png)

Reply:
Let me know if should be updated.